frontend/packages/react-app/src/reducers/tokens.ts

```diff
@@ -0,0 +1,37 @@
+import { TokenBalanceUserHolds, UserToken } from "../interfaces";
+import { ethers } from "ethers";
+
+export type ACTIONS =
+  | { type: "open_modal" }
+  | { type: "close_modal" }
+  | { type: "set_tokens"; payload: { data: TokenBalanceUserHolds } };
```

Comment:
Im not sure which style is good, but should be one of following as redux recommended: 
- Snake case: `OPEN_MODAL`
- "donamin:action": `modal:open`
- "domain/action": `modal/open`
https://redux.js.org/style-guide/style-guide#write-action-types-as-domaineventname

Reply:
Seems domain:action or domain:action is easy to write.

Reply:
Ok, let's use `domain:action` style.